internal/metrics/metrics.go

```diff
@@ -41,10 +44,17 @@ func New(options ...func(*Metrics) error) (Metrics, error) {
 		cpuInfoCmd:    setCommand("lscpu", "-J"),
 		gpuInfoCmd:    setCommand("lspci", "-n"),
 		archCmd:       setCommand("dpkg", "--print-architecture"),
+		libc6Cmd:      setCommand("apt-cache", "policy", "libc6"),
```

Comment:
I would rather using "dpkg" to detect libc6 version, which is easier to parse (tab separated fields) and already used in ubuntu-report.